src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -1030,7 +1030,7 @@ private void ImportDataMembers()
                         DataMember memberContract = new DataMember(member);
                         if (property != null)
                         {
-                            MethodInfo getMethod = property.GetMethod;
+                            MethodInfo getMethod = property.GetGetMethod();
                             if (getMethod == null || IsMethodOverriding(getMethod) || getMethod.GetParameters().Length > 0)
```

Comment:
>getMethod.GetParameters().Length > 0 [](start = 86, length = 36)

This doesn't event make sense as you are getting the method for the getter of the property, it can't have any parameters.